sdk/typespec/typespec_client_core/src/xml.rs

```diff
@@ -38,6 +41,7 @@ where
 }
 
 /// Serializes a type to bytes.
+/// Automatically includes the XML declaration.
```

Comment:
This is markdown, so if you wanted a separate line you have to insert a blank line (also below).

Reply:
Fixed.